### DISCUSSION TITLE
Question about Relative Dates

### DISCUSSION THREAD

**Discussion by jkenzer** [jamessimone/apex-rollup#119] [Q&A] [answered]
Salesforce SOQL queries support relative date queries like mentioned in [issue #83](https://github.com/jamessimone/apex-rollup/issues/83). I have a use case where I need to calculate roll-ups for the current fiscal year, previous fiscal year, and two fiscal years ago. Any advice on how to implement this given that #83 is still open?

Should I just do a greater than and less than in my Calc Item Where Clause for each of the date ranges and then update these yearly?

**Comment by jamessimone** [comment, marked as answer]
For now, I would use the relative date ranges as you were suggesting. I have a friend who was interested in working on #83 - let me check in with them to see if there’s been any progress on that, otherwise I’ll look to implement that feature by the end of this month!

**Reply by jamessimone** [reply]
@jkenzer with the release of `v1.3.9`, relative dates are now supported for use within Apex Rollup!

**Reply by jkenzer** [reply]
@jamessimone Awesome! Thanks for letting me know and thanks for the continued work on Apex Rollup.
